.github/workflows/docker_push.yml

```diff
@@ -54,7 +54,7 @@ jobs:
         uses: docker/metadata-action@v3
         with:
           images: |
-            ghcr.io/${{ github.repository_owner }}/flink-operator
+            ghcr.io/${{ github.repository }}
```

Comment:
why was /flink-operator removed here?

Reply:
`github.repository_owner` == `apache`
`github.repository` == `apache/flink-kubernetes-operator`

In short now that the image name equals the repo name I can write this shorter.